### PR TITLE
docs: added documentation for query-service local setup

### PR DESCRIPTION
**Pull request by jshiwam**
Updated Query-Service documentation for local setup.

### CODE REVIEW

File: pkg/query-service/README.md
Comment (makeavish) [inline pkg/query-service/README.md:10]: @jshiwam This link shouldn't be pointing to your fork.

Comment (jshiwam) [reply pkg/query-service/README.md:10]: Will fix it and update the PR.

Comment (jshiwam) [reply pkg/query-service/README.md]: Done!!

File: pkg/query-service/README.md
Comment (palashgdev) [inline pkg/query-service/README.md:69]: please use from env here

**Review by palashgdev — CHANGES_REQUESTED** [review]

File: pkg/query-service/README.md
Comment (jshiwam) [reply pkg/query-service/README.md:69]: If you want me to add it here as follows
```
export const ENVIRONMENT = {
	baseURL:
		process?.env?.FRONTEND_API_ENDPOINT ||
		process?.env?.FRONTEND_ALERTMANAGER_ENDPOINT ||
                process?.env?.GITPOD_WORKSPACE_URL?.replace('://', '://8080-') ||
		'',
};

```
where FRONTEND_ALERTMANAGER_ENDPOINT=http://localhost:9093/
Then in that case it won't be picked up. Should I create another variable instead of used baseURL then?

**Review by makeavish — APPROVED** [review]

**Comment by makeavish** [comment]
LGTM, can be merged after @palashgdev 's approval

File: pkg/query-service/README.md
Comment (palashgdev) [reply pkg/query-service/README.md:69]: No another env is not feasible

**Review by palashgdev — APPROVED** [review]
LGTM

**Pull request merged**
